content/en/opentelemetry/guide/otlp_delta_temporality.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Producing delta temporality metrics with OpenTelemetry
+kind: guide
+further_reading:
+- link: "/metrics/open_telemetry/otlp_metric_types"
+  tag: "Documentation"
+  text: "OTLP Metric Types"
+- link: "/opentelemetry/"
+  tag: "Documentation"
+  text: "OpenTelemetry Support in Datadog"
+---
+
+## Overview
+
+The OpenTelemetry protocol (OTLP) can send [several metric types][1] some of which can have *delta* of *cumulative* [aggregation temporality][2]. Datadog products work best with delta aggregation temporality for monotonic sums, histograms, and exponential histograms. You can select which aggregation temporality to export your metrics with on your OpenTelemetry SDK or use the [OpenTelemetry Collector `cumulativetodelta` processor][3].
```

Comment:
is it *delta* or *cumulative* ?

Reply:
I assume you are talking about
> Datadog products work best with delta aggregation temporality for monotonic sums, histograms, and exponential histograms 

? 
If so, delta is the right word here: our metrics model is delta-based, so using delta makes it so that we don't have to transform the data in the exporter/Agent.

Reply:
I think Dinesh means that "delta of cumulative" should be "delta or cumulative". I am editing it thusly.

Reply:
Ah right, yes, good catch @dineshg13 and @kayayarai :)